docs/excel/excel-data-types-overview.md

```diff
@@ -12,23 +12,26 @@ ms.localizationpriority: high
 
 Data types organize complex data structures as objects. This includes formatted number values, web images, and entities as [entity cards](excel-data-types-entity-card.md).
 
+The following screenshot highlights one of the primary features of data types: an entity card. In this case, the entity card shows expanded information about the **Tofu** product from a list of grocery store products.
+
+:::image type="content" source="../images/excel-data-types-entity-card-tofu.png" alt-text="A screenshot showing an entity value data type with the card window displayed.":::
+
 Prior to the data types addition, the Excel JavaScript API supported string, number, boolean, and error data types. The Excel UI formatting layer is capable of adding currency, date, and other types of formatting to cells that contain the four original data types, but this formatting layer only controls the display of the original data types in the Excel UI. The underlying number value is not changed, even when a cell in the Excel UI is formatted as currency or a date. This gap between an underlying value and the formatted display in the Excel UI can result in confusion and errors during add-in calculations. The data types APIs are a solution to this gap.
 
 Data types expand Excel JavaScript API support beyond the four original data types (string, number, boolean, and error) to include [web images](excel-data-types-concepts.md#web-image-values), [formatted number values](excel-data-types-concepts.md#formatted-number-values), [entities](excel-data-types-concepts.md#entity-values), arrays within entities, and improved [error data types](excel-data-types-concepts.md#improved-error-support) as flexible data structures. These types, which power many [linked data types](https://support.microsoft.com/office/what-linked-data-types-are-available-in-excel-6510ab58-52f6-4368-ba0f-6a76c0190772) experiences, allow for precision and simplicity during add-in calculations and extend the potential of Excel add-ins beyond a 2-dimensional grid.
 
-To learn how to use data types APIs, start with the [Excel data types core concepts](excel-data-types-concepts.md) article.
+## Resources
+
+Explore data types with the following resources.
+
+1. Learn data types basics in the [Excel data types core concepts](excel-data-types-concepts.md) article.
+1. Install [Script Lab](../overview/explore-with-script-lab.md) in Excel and explore the **Data types** section in our **Samples** library.
+1. Learn how to extend Excel beyond a 2-dimensional grid with entity data types in the [Use cards with entity value data types](excel-data-types-entity-card.md) article.
+1. Try the [Create and explore data types in Excel](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/excel-data-types-explorer) sample in our [OfficeDev/Office-Add-in-samples](https://github.com/OfficeDev/Office-Add-in-samples) repository.
 
 > [!NOTE]
```

Comment:
This note is a repeat of bullet 2.

Reply:
I'm hesitant to remove the note because I originally added it in response to feedback (uncertainty about how to get started developing data types). I think the point of the note in this location is to emphasize that the reader can skip step 1 and move straight to Script Lab, if they prefer a sandbox type learning style. I agree that it's somewhat redundant, though. What do you think?

Reply:
What if the note is moved right after the first paragraph ("skip the reading and try it out") or after the picture (so it'll be visually inline with the image)?

Reply:
Done! Moved it to after the image. Thanks for all of the suggestions!